src/pages/guides/using/marketo/marketo-observability-data-stream-setup.md

```diff
@@ -0,0 +1,291 @@
+---
+title: Setting up the Marketo Observability Data Stream
+---
+
+# Setting up the Marketo Observability Data Stream
+
+These instructions describe how to set up and get started using Adobe I/O Events to subscribe to Marketo Observability events.
+
+- [Introduction](#introduction)
+- [Pre-Requisite Setup](#pre-requisite-setup)
+- [Setup Adobe I/O](#setup-adobe-i/o)
```

Comment:
This link does not work because the title has been changed. Please update this to ```- [Setup Adobe I/O](#getting-started-with-adobe-io)```

Reply:
Yup, I think I updated the section but forgot to update the link to it, good catch.